js/steps.js

```diff
@@ -0,0 +1,143 @@
+class Steps {
+  constructor({ selector = null, steps = null, defaultValidator = false, validateInOrder = true, navigateOnClick = !validateInOrder } = {}) {
+    this.rootElement = document.querySelector(selector);
+    this.steps = steps;
+    this.childElements = [];
+    this.navigateOnClick = navigateOnClick;
+    this.validateInOrder = validateInOrder;
+    this.defaultValidator = defaultValidator;
+    this.createAllHTMLElements();
+    this.changeActiveStep(0);
+  }
+  changeActiveStep(stepIndex) {
+    this.activeStep = stepIndex;
+    this.childElements.map(childElement => childElement.stepContainer.classList.remove("active"));
+    this.childElements[stepIndex].stepContainer.classList.add("active");
+    this.changeProgressBarWidth(this.childElements[stepIndex].stepContainer);
+  }
+  completeStep(isValid) {
+    let classValue = isValid ? "valid" : "invalid";
+    this.CurrentElement.stepContainer.classList.remove("valid");
+    this.CurrentElement.stepContainer.classList.remove("invalid");
```

Comment:
you can remove more classes at the same time with .remove()

ex. myDiv.classList.remove( "flex", "column" )